soc/arm/st_stm32/stm32wb/Kconfig.defconfig.series

```diff
@@ -32,4 +32,11 @@ config I2C_STM32_V2
 
 endif # I2C_STM32
 
+if ADC
```

Comment:
Already done in `soc/arm/st_stm32/common/Kconfig.defconfig.series`.
You can remove this  change